modules/KalturaSupport/resources/mw.KBaseMediaList.js

```diff
@@ -268,7 +268,7 @@
 		//Media Item
 		renderMediaList: function(callback){
 			//Only render if medialist item are present
-			if (this.getTemplateData().length > 0) {
+			if (this.getTemplateData().length > (0 - !this.getConfig('showEmptyPlaylistError'))) {
 				//Generate new list template data
```

Comment:
@eitanavgil  - I don't understand what you are trying to do here: the expression in brackets will always be 0 or -1.